k8s/staging/runners/protected/windows/x86_64/v2/release.yaml

```diff
@@ -57,27 +57,14 @@ spec:
           pre_build_script = """
           Write-Output 'Executing Spack pre-build setup script'
 
-          $PY3 = $null
-          $pythonCmds = @('python3', 'python')
-
-          foreach ($cmd in $pythonCmds) {
-              if (Get-Command -ErrorAction SilentlyContinue $cmd) {
-                  $PY3 = Get-Command $cmd
-                  break
-              }
-          }
-
-          if (-not $PY3) {
```

Comment:
I think it is still good to have a nice error message if python is missing.

Reply:
Yeah, good point, was probably a bit overzealous with the cleanup.

Reply:
Added this back.